docker/acra-build.dockerfile

```diff
@@ -73,7 +73,7 @@ ENV PATH="$PATH:/usr/local/lib/go/latest/bin"
 ENV PATH_ACRA="/acra"
 COPY ./ "${PATH_ACRA}/"
 # Fetch all dependencies and build all binaries in acra
-RUN cd "${PATH_ACRA}" && go install ./cmd/...
+RUN cd "${PATH_ACRA}" && go install -tags netgo ./cmd/...
```

Comment:
@shadinua ^

Reply:
Need additional testing from @shadinua

Reply:
Tested with scratch and Debian-based images. Thanks!